bin/gimme-aws-creds

```diff
@@ -141,48 +261,73 @@ class GimmeAWSCreds(object):
 
         # get the config dict
         conf_dict = config.get_config_dict()
-        config.get_user_creds()
 
-        if conf_dict['idp_entry_url'] in [None, '']:
-            print('No IDP entry URL in configuration.  Try running --config again.')
+        if conf_dict['okta_org_url'] in [None, '']:
+            print('No Okta organization URL in configuration.  Try running --config again.')
+            sys.exit(1)
 
-        idp_entry_url = conf_dict['idp_entry_url'] + '/api/v1'
+        okta = OktaClient(conf_dict['okta_org_url'], config.verify_ssl_certs)
+        if config.username is not None:
+            okta.set_username(config.username)
 
-        # create otka client
-        api_key = self._get_okta_api_key(
-            config.username,
-            config.password,
-            idp_entry_url,
-            cerberus_url=conf_dict['cerberus_url']
-        )
+        # Call the Okta APIs and proces data locally
+        if conf_dict['gimme_creds_server'] == 'internal':
+            # Okta API key is required when calling Okta APIs internally
+            if config.api_key is None:
+                print('OKTA_API_KEY environment variable not found!')
+                sys.exit(1)
+            # Authenticate with Okta
+            auth_result = okta.auth_session()
+
+            print("Authentication Success! Getting AWS Accounts", end='', flush=True)
+            aws_results = self._get_aws_account_info(conf_dict['okta_org_url'], config.api_key, auth_result['username'])
+
+        # Use the gimme_creds_lambda service
+        else:
+            if conf_dict['gimme_creds_server'] in [None, '']:
```

Comment:
These if statements should read like the following:
`if not conf_dict.get('gimme_creds_server'):`

A response of an empty string or `None` will both evaluate to false.  Also, using `.get()` makes the existence of the key optional; if not found, it will return `None` instead of throwing a `KeyError`.

Reply:
The check for gimme_creds_server can actually be moved up to before this if/else statement, since it's used in both conditions.  We need that key to exist in some form in order to continue.